tests/test_xsd.py

```diff
@@ -181,16 +217,20 @@ def test_typo_property_asc(testdir, properly_decorated_test_function):
 
     # Add another property element for the testcase.
     xml_doc.find('./testcase/properties/property').attrib['name'] = 'wrong_test_id'
-    xmlschema = etree.XMLSchema(etree.parse(get_xsd('asc')))
+    xmlschema = etree.XMLSchema(etree.parse(xmlpf._get_xsd('asc')))
```

Comment:
Since an external program is accessing this method it should be marked as public.